storage/lmdb/src/lib.rs

```diff
@@ -142,6 +145,10 @@ impl LmdbImpl {
         })
     }
 
+    // FIXME: I'd like to replace the SeqCst ordering here with Release, because it looks
```

Comment:
It's designed to be an overkill. I don't mind though. Feel free to change it to release.